webapp/lib/app/nook/view.dart

```diff
@@ -101,23 +105,26 @@ class NookPageView extends PageView {
     showNormalStatus('signed out');
   }
 
-  var layouts = {
-    'true-true':   {"conversationListPanel": "w-20", "conversationPanel": "w-40", "replyPanel": "w-25", "tagPanel": "w-15"},
-    'true-false':  {"conversationListPanel": "w-25", "conversationPanel": "w-45", "replyPanel": "w-30", "tagPanel": "w-0" },
-    'false-true':  {"conversationListPanel": "w-30", "conversationPanel": "w-50", "replyPanel": "w-0" , "tagPanel": "w-20"},
-    'false-false': {"conversationListPanel": "w-35", "conversationPanel": "w-65", "replyPanel": "w-0" , "tagPanel": "w-0" },
-  };
+  void showPanels(showReplyPanel, enableEditNotesPanel, showTagPanel) {
+    List<TabView> tabsToSet = [];
 
-  var layout = ['1', '1'];
+    if(showReplyPanel) {
+      var standardMessageTab = TabView('standard_message', "Standard messages", replyPanelView.replyPanel);
+      tabsToSet.add(standardMessageTab);
+    }
 
-  void showPanels(showReplyPanel, showEditNotesPanel, showTagPanel) {
-    var showReplyNotesPanel = showReplyPanel || showEditNotesPanel;
-    replyPanelView.replyPanel.classes.toggle('hidden', !showReplyNotesPanel);
-    showReplyPanel ? replyPanelView.enableReplies() : replyPanelView.disableReplies();
-    replyPanelView.enableEditableNotes(showEditNotesPanel);
-    tagPanelView.tagPanel.classes.toggle('hidden', !showTagPanel);
+    if(enableEditNotesPanel) {
+      var notesTab = TabView('notes', "Notes", notesPanelView.notesPanel);
+      tabsToSet.add(notesTab);
```

Comment:
I think this adds the notes tab in between the standard messages and the tags panels (ie. 2nd place) - can we have the notes as the third tab instead?